ui/pages/first-time-flow/seed-phrase/reveal-seed-phrase/reveal-seed-phrase.component.js

```diff
@@ -112,7 +112,11 @@ export default class RevealSeedPhrase extends PureComponent {
               this.setState({ isShowingSeedPhrase: true });
             }}
           >
-            <LockIcon width="28px" height="35px" fill="#FFFFFF" />
+            <LockIcon
+              width="28px"
+              height="35px"
+              fill="var(--color-background-default)"
```

Comment:
If this icon is on top of the secret blocker it should use overlay-inverse
```suggestion
              fill="var(--color-overlay-inverse)"
```